lib/packwerk/generators/binstub.rb

```diff
@@ -0,0 +1,42 @@
+# typed: true
+# frozen_string_literal: true
+
+module Packwerk
+  module Generators
+    class Binstub
+      class << self
+        def generate(root: ".", out: $stdout)
+          new(root, out: out).generate
+        end
+      end
+
+      def initialize(root, out: $stdout)
+        @root = root
+        @out = out
+      end
+
+      def generate
+        @out.puts("📦 Generating binstub...")
+        generate_packwerk_validate_script
+      end
+
+      private
+
+      def generate_packwerk_validate_script
```

Comment:
this is not part of the revert, you added this, right? I wonder why we didn't need this before.

We also had the `SpringCommand` before, which I'm not quite sure what it does 🤷🏼

Reply:
It was part of the revert:
It was called `ApplicationValidation`, which was generated when initializing packwerk here: https://github.com/Shopify/packwerk/pull/128/files#diff-ac5ae86965a90253f8d378798aa32f4041ec009d6b79c4420e88be6e67108e88L80

That application validation generation copied the binstub here: https://github.com/Shopify/packwerk/pull/128/files#diff-5cc7180b28a34feeec40fbab8714e5bb0dd61d6db8f1554084f63f9d2ca1e9b2L39

I just called it a `binstub` because that's really its main point. Before it was called `ApplicationValidation` I think because it was intended to be what you called when validating the application, but now it's what we will expect people to call always regardless of if validating or if running `bin/packwerk update-deprecations`, etc.